airflow/providers/airbyte/operators/airbyte.py

```diff
@@ -144,3 +147,30 @@ def on_kill(self):
         if self.job_id:
             self.log.info("on_kill: cancel the airbyte Job %s", self.job_id)
             hook.cancel_job(self.job_id)
+
+    def get_openlineage_facets_on_start(self):
```

Comment:
Shouldn't we implement on_complete to also have the job_id ? Can we somehow get more information when using that id?

Reply:
I ve changed implementation on complete (it makes more sense to keep the ingestion at the end). Hmm I think all the information which we can get is in the connection endpoint, when sync starts the connections can't change. Or you mean data like number of records transmitted I don't know if its a part of open lineage specs.